GCAppirater.m

```diff
@@ -653,4 +573,27 @@ - (UIAlertController *)getRateAlertController {
   return alert;
 }
 
+- (BOOL)userIsEligibleToRate {
+  NSDate *nextRatingDate = [[NSUserDefaults standardUserDefaults] objectForKey:kGCAppiraterAskAgainDate];
+  if ( [nextRatingDate isEqual:[NSDate distantFuture]] ) {
```

Comment:
This check is redundant with the final else. If for some reason apple were to change `distantFuture`, this check would fail, but the compare on 583 would still do the right thing.